lock/lock.go

```diff
@@ -314,6 +315,50 @@ func GetLockStatus(options *Options) (*dynamodb.GetItemOutput, error) {
 	return item, nil
 }
 
+type DeploymentLock struct {
+	// In the DynamoDB lock table, "LockID" will be the key and the deployment URL will be the value
+	DeploymentURL string `json:"LockID"`
+}
+
+// ScanTable will perform a scan operation on the indicated DynamoDB table
+// This operation is useful in certain cases, for example when we want to
+// generate a report of all currently active ref arch deployments tracked in our lock table
```

Comment:
update docs, and mention what the output string is (`list of lock IDs on the table that are currently held`?)

Reply:
Fixed in [ea948a1](https://github.com/gruntwork-io/go-commons/pull/60/commits/ea948a1aca5a5c3b4fc64e48c9ec057f0321530d)